docs/Makefile

```diff
@@ -1,39 +1,26 @@
 # Execute commands regardless of physical state of files
-.PHONY: all build preview sphinx
+.PHONY: all build preview sphinx d2-fmt
 
 # Overall build scheme
-all: models.pdf \
-	models.png \
-	simplified-models.pdf \
-	simplified-models.png \
-	high-level.pdf \
-	high-level.png \
-	question-flow.pdf \
-	question-flow.png \
-	build
+all: build
+
+# fmt doesn't check if the formatting changed, it just formats everything
```

Comment:
Oof. I see https://github.com/terrastruct/d2/issues/1180. Can we link to that? I might try opening a PR to add support for that.

Reply:
Linked